elements/lisk-codec/src/boolean.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+/* eslint-disable no-bitwise */
+/* eslint-disable no-param-reassign */
+
+
+interface SchemaProperty {
+	readonly dataType: string;
+}
+
+export const writeBoolean = (value: boolean, _schema: SchemaProperty): Buffer =>
+    value ? Buffer.from('01', 'hex') : Buffer.from('00', 'hex');
+
+export const readBoolean = (buffer: Buffer, _schema: SchemaProperty): boolean =>
+	buffer[0] === 0x01;
```

Comment:
I think that here it is more common to have the function return `!(buf===0x00)`.
With the current formula, only `0x01` will return true.

Reply:
true! will change them